src/components/ContactUs/FormContactUs/FormContactUs.tsx

```diff
@@ -8,48 +8,13 @@ import { validationSchema } from './FormContactUs.yup'
 import useIntlMessages from 'hooks/useIntlMessages'
 import Input from 'ui/input/Input'
 import Textarea from 'ui/Textarea/Textarea'
-import {
-  Error,
-  FormWrapper,
-  ContainerForm,
-  ButtonForm,
-} from './FormContactUs.styles'
+import { FormWrapper, ContainerForm, ButtonForm } from './FormContactUs.styles'
 import MultiSelect from 'ui/MultiSelect/MultiSelect'
-
-interface IValues {
-  fullName: string
-  country: string
-  email: string
-  message: string
-}
-
-const initialValues: IValues = {
-  fullName: '',
-  country: '',
-  email: '',
-  message: '',
-}
-
-interface IServiceOptions {
-  key: string
-  name: string
-}
-
-const servicesOptions: IServiceOptions[] = [
-  { key: '1', name: 'Development of graphic pieces' },
-  { key: '2', name: 'Branding' },
-  { key: '3', name: 'UX/UI Design' },
-  { key: '4', name: 'UX Writing' },
-  { key: '5', name: 'Web development' },
-  { key: '6', name: 'Full stack applications' },
-  { key: '7', name: 'Multiplatform applications' },
-  { key: '8', name: 'Mobile Development - iOS, Android, Flutter' },
-  { key: '9', name: 'E-commerce' },
-  { key: '10', name: 'Cloud migrations and infrastructure' },
-  { key: '11', name: 'Digital staff' },
-  { key: '12', name: 'SEO Optimization/ SEM' },
-  { key: '13', name: 'Digital Marketing' },
-]
+import { servicesOptions } from './FormData'
+import { IValues } from 'types/ui/Form'
+import { IServiceOptions } from 'types/ui/Multiselect'
+import { MdError } from 'react-icons/md'
```

Comment:
Third party library, this must be above the file, separated by a line.

```
import third party library
/* space */
import our files
```

Reply:
done